crates/telio-model/src/features.rs

```diff
@@ -154,6 +154,9 @@ pub struct FeatureNurse {
     /// Enable/disable NAT-traversal connections data. Enabled by default.
     #[serde(default = "FeatureNurse::default_enable_nat_traversal_conn_data")]
     pub enable_nat_traversal_conn_data: bool,
+    /// How long a session can be before it is forcibly reported, in seconds. Default value is 24h.
+    #[serde(default = "FeatureNurse::default_state_duration_cap")]
+    pub state_duration_cap: u64,
```

Comment:
If the Duration is not possible, I would make a typedef of `DurationCapSecs` or similar ...

Reply:
I see where you're coming from but the `duration` in the name doesn't refer to the type duration but rather the non-programming concept of duration, so I don't think this needs a typedef, especially not with the doc comment being there